APIProject/NetWorking/ApiManager.swift

```diff
@@ -3,63 +3,28 @@
 //  APIProject
 //
 //  Created by User-P2 on 8/26/22.
-//https://youtu.be/Uokamj2WtxU?t=1000
-
-import Foundation
 import UIKit
-import SDWebImage
-protocol managerDelegate {
-    func showModelList(list: [Character])
+
+protocol ManagerDelegate {
+    func showModelList(characters: [Character])
 }
 
-class ApiManager {
-    
-    var delegate: managerDelegate?
+final public class ApiManager {
+    var delegate: ManagerDelegate?
     
     func getData() {
-        let urlString = URL(string: "https://bobsburgers-api.herokuapp.com/characters/?limit=9&skip=6")
-        URLSession.shared.dataTask(with: urlString!) { urlData, urlResponse, urlError in
+        guard let url = URL(string: "https://bobsburgers-api.herokuapp.com/characters/?limit=9&skip=6") else { return }
+        URLSession.shared.dataTask(with: url) { urlData, urlResponse, urlError in
```

Comment:
Remover el prefijo url a estas variables
` URLSession.shared.dataTask(with: url) { data, response, error in`

Si no utilizas una variable la puedes marcar con un _ 
`{ data, _, error in`